tool/tctl/common/user_command.go

```diff
@@ -475,6 +482,12 @@ func (u *UserCommand) Update(ctx context.Context, client *authclient.Client) err
 		updateMessages["Host user GID"] = []string{u.hostUserGID}
 	}
 
+	if len(u.allowedMCPTools) > 0 {
+		mcpTools := flattenSlice(u.allowedMCPTools)
```

Comment:
wouldn't you flatten first and only if the length of return value is not 0 do the this setup?

Reply:
as mentioned in the other comment, i reverted the empty check in flattenSlice. so logic is consistent now with existing traits.